src/main/resources/archetype-resources/src/main/resources/config/__rootArtifactId__/mycore.properties.template

```diff
@@ -0,0 +1,82 @@
+# DOI configuration                                                          #
+##############################################################################
+# Configure local or in ansible inventory
+MCR.DOI.Prefix=
+MCR.PI.Service.Datacite.RegisterBaseURL=
+MCR.PI.Service.Datacite.Username=
+MCR.PI.Service.Datacite.Password=
+
+MCR.DOI.HostingInstitution=
+# Uncomment only to activate DOI
+#MCR.PI.Service.Datacite=org.mycore.pi.doi.MCRDOIService
+MCR.PI.Service.Datacite.MetadataService=MODSDOI
+
+# Does that DOI is only set after document is published, should be standard behavior
+MCR.PI.Service.Datacite.RegistrationPredicate=org.mycore.pi.condition.MCRPIPublishedPredicate
+MCR.PI.Service.Datacite.Transformer=datacite
+MCR.PI.Service.Datacite.Generator=org.mycore.pi.MCRGenericPIGenerator
+
+MCR.PI.Generator.Datacite.GeneralPattern=%MCR.DOI.Prefix%/$ObjectDate-$Count
+MCR.PI.Generator.Datacite.DateFormat=yyyy
+MCR.PI.Generator.Datacite.CountPrecision=3
+
+MCR.PI.MetadataService.MODSDOI=org.mycore.mods.identifier.MCRMODSDOIMetadataService
+MCR.PI.MetadataService.MODSDOI.Type=doi
+MCR.PI.Parsers.doi=org.mycore.pi.doi.MCRDOIParser
+
+##############################################################################
+# OAI Data Provider                                                          #
+##############################################################################
+MCR.OAIDataProvider.OAI2.RepositoryName=
+MCR.OAIDataProvider.OAI2.RepositoryIdentifier=
+MCR.OAIDataProvider.OAI2.AdminEmail=
+MCR.OAIDataProvider.OAI2.RecordSampleID=
+MCR.OAIDataProvider.OAI2.DescriptionURI.EPrints=webapp:oai/oai-eprints.xml
+MCR.OAIDataProvider.OAI2.DescriptionURI.Rights=webapp:oai/oai-rights.xml
+
+##############################################################################
+# MIR ePuSta                                                                 #
+##############################################################################
+# show ePuSta Statistics in metadata view ('show', 'hide')
+MIR.ePuSta=hide
+MIR.ePuSta.GraphProviderURL=
+MIR.ePuSta.Prefix=
+
+##############################################################################
+# Workflow                                                                   #
+##############################################################################
+MIR.Workflow.Menu=true
+MIR.Workflow.Box=true
+
+##############################################################################
+# Editor                                                                     #
+##############################################################################
+# Customization of XEditor forms
+MIR.EditorForms.CustomIncludes=%MIR.EditorForms.CustomIncludes%,xslStyle:editor/mir2xeditor:webapp:editor/editor-customization.xed
+
+##############################################################################
+# Error Page                                                                 #
```

Comment:
Perhaps we could generalize the name for the Error Page section a little more, because a section seems like overkill to me. Perhaps we could add or combine it with Layout and call it Frontend.